src/helpers/referenceHelpers.js

```diff
@@ -16,7 +16,7 @@ const RANGE_SEPARATORS = [
  * takes a reference and splits into individual verses or verse spans.
  * @param {string} ref - reference in format such as:
  *   “2:4-5”, “2:3a”, “2-3b-4a”, “2:7,12”, “7:11-8:2”, "6:15-16;7:2"
- * @return {[{chapter, verse, endChapter, endVerse}]}
+ * @return {Object}  The Verse Chunk returned
```

Comment:
Please investigate other ways to show more detail than just object type

Reply:
> Please investigate other ways to show more detail than just object type

I guess this has to do with some limitation with the jsdoc API generator, which I have encountered before. If that's the case, what I do is create a typedef and use that to define the type of the return: https://github.com/Proskomma/epitelete/blob/f2d20d5cd30270920b053c00c796df68f93b20b5/src/index.js#L622

That will make the generator not get confused.